reporter-web-app/config/webpack.config.js

```diff
@@ -649,7 +649,10 @@ module.exports = function(webpackEnv) {
               return console.log(err);
             }
 
-            const result = data.replace(/(<script.*? id="ort-report-data">)([\s\S]*?)(<\/script>)/g, '$1$3');
+            const result = data.replace(
+                /(<script.*? id="ort-report-data">)([\s\S]*?)(<\/script>)/g,
```

Comment:
BTW, what kind of weird regex is `[\s\S]*?`, as it matches both space and non-space characters, including an empty string, i.e. everything?

Reply:
I did not think about it, but I presume it's this issue with newlines not being matched by `.*`:
https://stackoverflow.com/a/4544642/639421

Reply:
Interesting, thanks. https://stackoverflow.com/a/16119722/1127485 has some more details.